soap/src/java/com/zimbra/soap/admin/type/DataSourceType.java

```diff
@@ -26,7 +26,7 @@
 @XmlEnum
 public enum DataSourceType {
     // case must match protocol
-    pop3, imap, caldav, contacts, yab, rss, cal, gal, xsync, tagmap;
+    pop3, imap, caldav, contacts, yab, rss, cal, gal, xsync, tagmap, custom;
```

Comment:
Is there going to be some description somewhere of exactly what custom means?

Reply:
once I get a working version of Yahoo (or any other) oauth-based data source, I'll add a wiki about datasources. So far it's not useful for anyone